src/jit/codegenxarch.cpp

```diff
@@ -2645,8 +2576,6 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
 bool
 CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
 {
-
-
     if (varTypeIsStruct(treeNode))
```

Comment:
As i explained to offiline, this is not the same as #2700.  Please open a separate issue for modelling return type of GT_CALL nodes that return structs in two return registers.